Description: Build system for shared library.
Author: Scott Howard <showard@debian.org>

Index: dxflib/Makefile.am
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ dxflib/Makefile.am	2011-06-19 13:12:27.730403356 -0400
@@ -0,0 +1,32 @@
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = dxflib.pc
+
+BASE_DIR = ./src
+
+BASE_SRC = \
+		$(BASE_DIR)/dl_dxf.cpp \
+		$(BASE_DIR)/dl_writer_ascii.cpp
+
+BASE_INS = \
+		$(BASE_DIR)/dl_attributes.h \
+		$(BASE_DIR)/dl_codes.h \
+		$(BASE_DIR)/dl_creationadapter.h \
+		$(BASE_DIR)/dl_creationinterface.h \
+		$(BASE_DIR)/dl_dxf.h \
+		$(BASE_DIR)/dl_entities.h \
+		$(BASE_DIR)/dl_exception.h \
+		$(BASE_DIR)/dl_extrusion.h \
+		$(BASE_DIR)/dl_writer.h \
+		$(BASE_DIR)/dl_writer_ascii.h
+
+lib_LTLIBRARIES = libdxflib.la
+libdxflib_la_SOURCES = $(BASE_SRC)
+libdxflib_la_includedir=$(includedir)/dxflib
+libdxflib_la_include_HEADERS = $(BASE_INS)
+libdxflib_la_LDFLAGS = -release @VERSION@
+
+testing:
+	(cd ./test ; make)
+
+docu:
+	-(doxygen ./doxygen.cfg)
Index: dxflib/dxflib.pc.in
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ dxflib/dxflib.pc.in	2011-06-19 13:17:12.712300083 -0400
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@/dxflib
+
+Name: dxflib
+Description: Library for reading dxf files
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -ldxflib
+Cflags: -I${includedir}
Index: dxflib/configure.in
===================================================================
--- dxflib.orig/configure.in	2011-06-19 12:59:06.825498515 -0400
+++ dxflib/configure.in	2011-06-19 12:59:12.835085646 -0400
@@ -2,7 +2,11 @@
 #
 # Check for one project file
 #
-AC_INIT(src/dl_dxf.cpp)
+AC_PREREQ([2.67])
+AC_INIT([dxflib], [2.2.0.0], [])
+AC_CONFIG_SRCDIR([src/dl_exception.h])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AC_PROG_LIBTOOL
 
 PRODUCT=dxflib
 
@@ -48,7 +52,7 @@
 # finally create makefiles using Makefile.in
 #
 echo
-AC_OUTPUT(Makefile test/Makefile)
+AC_OUTPUT(Makefile test/Makefile dxflib.pc)
 
 echo
 echo "Run 'make depend' to create dependencies."
