Description: Build system for shared library.
Author: Scott Howard <showard@debian.org>

--- /dev/null
+++ dxflib-2.2.0.0/Makefile.am
@@ -0,0 +1,31 @@
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = dxflib.pc
+
+BASE_DIR = ./src
+
+BASE_SRC = \
+		$(BASE_DIR)/dl_dxf.cpp \
+		$(BASE_DIR)/dl_writer_ascii.cpp
+
+BASE_INS = \
+		$(BASE_DIR)/dl_attributes.h \
+		$(BASE_DIR)/dl_codes.h \
+		$(BASE_DIR)/dl_creationadapter.h \
+		$(BASE_DIR)/dl_creationinterface.h \
+		$(BASE_DIR)/dl_dxf.h \
+		$(BASE_DIR)/dl_entities.h \
+		$(BASE_DIR)/dl_exception.h \
+		$(BASE_DIR)/dl_extrusion.h \
+		$(BASE_DIR)/dl_writer.h \
+		$(BASE_DIR)/dl_writer_ascii.h
+
+lib_LTLIBRARIES = libdxflib.la
+libdxflib_la_SOURCES = $(BASE_SRC)
+include_HEADERS = $(BASE_INS)
+libdxflib_la_LDFLAGS = -release @VERSION@
+
+testing:
+	(cd ./test ; make)
+
+docu:
+	-(doxygen ./doxygen.cfg)
--- /dev/null
+++ dxflib-2.2.0.0/dxflib.pc.in
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: dxflib
+Description: Library for reading dxf files
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -ldxflib
+Cflags: -I${includedir}
--- dxflib-2.2.0.0.orig/configure.in
+++ dxflib-2.2.0.0/configure.in
@@ -2,7 +2,11 @@
 #
 # Check for one project file
 #
-AC_INIT(src/dl_dxf.cpp)
+AC_PREREQ([2.67])
+AC_INIT([dxflib], [2.2.0.0], [])
+AC_CONFIG_SRCDIR([src/dl_exception.h])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AC_PROG_LIBTOOL
 
 PRODUCT=dxflib
 
@@ -48,7 +52,7 @@ AC_CHECK_HEADERS(limits.h)
 # finally create makefiles using Makefile.in
 #
 echo
-AC_OUTPUT(Makefile test/Makefile)
+AC_OUTPUT(Makefile test/Makefile dxflib.pc)
 
 echo
 echo "Run 'make depend' to create dependencies."
