Author: Alastair McKinstry <mckinstry@debian.org>
Description: Re-include configure.in missing from 2.12.2
Forwarded: no
Last-Updated: 2016-01-04

Index: dxflib-3.12.2/configure.in
===================================================================
--- /dev/null
+++ dxflib-3.12.2/configure.in
@@ -0,0 +1,60 @@
+
+#
+# Check for one project file
+#
+AC_PREREQ([2.67])
+AC_INIT([dxflib], [2.12.2], [])
+AC_CONFIG_SRCDIR([src/dl_exception.h])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
+AC_PROG_LIBTOOL
+
+PRODUCT=dxflib
+
+#
+# check cannonical system name
+#
+AC_CANONICAL_HOST
+case "$host" in
+*-linux-gnu )  AC_DEFINE(LINUX) ;;
+     *-aix* )  AC_DEFINE(AIX) ;;
+          * )  AC_DEFINE(UNIX) ;;
+esac
+
+#
+# check C compiler, preprocesor, etc.
+#
+AC_PROG_CC
+AC_PROG_CPP
+AC_PROG_CXX
+AC_PROG_INSTALL
+AC_CHECK_PROG(FIND, find, find, :)
+AC_CHECK_PROG(MAKEDEPEND, makedepend, makedepend, :)
+
+#
+# Try to locate the X Window System include files and libraries
+# and add /usr/local to include and lib path and add -lm (for testing)
+#
+AC_PATH_XTRA
+CFLAGS="$CFLAGS $X_CFLAGS"
+LDFLAGS="$LDFLAGS $X_LIBS -L/usr/local/lib"
+
+
+#
+# Declare variables which we want substituted in the Makefile.in's
+#
+
+
+AC_HEADER_STDC
+AC_CHECK_HEADERS(limits.h)
+
+
+#
+# finally create makefiles using Makefile.in
+#
+echo
+AC_OUTPUT(Makefile  dxflib.pc)
+
+echo
+echo "Run 'make depend' to create dependencies."
+echo
