Description: Missing include needs to be added
Author: Scott Howard <showard@debian.org>

Index: dxflib/src/dl_writer.h
===================================================================
--- dxflib.orig/src/dl_writer.h	2013-06-23 13:13:35.127629322 -0400
+++ dxflib/src/dl_writer.h	2013-06-23 13:13:35.119629321 -0400
@@ -37,6 +37,7 @@
 #endif
 
 #include <iostream>
+#include <cstring>
 
 #include "dl_attributes.h"
 #include "dl_codes.h"
