Description: Missing include needs to be added
Author: Scott Howard <showard@debian.org>

Index: dxflib-3.7.5/src/dl_writer.h
===================================================================
--- dxflib-3.7.5.orig/src/dl_writer.h
+++ dxflib-3.7.5/src/dl_writer.h
@@ -37,6 +37,7 @@
 #endif // _MSC_VER > 1000
 
 #include <iostream>
+#include <cstring>
 #include <algorithm>
 
 #include "dl_attributes.h"
